### DISCUSSION TITLE
compatible for Seeed Studio XIAO ESP32S3 Sense with ESP32S3 ?

### DISCUSSION THREAD

**Discussion by Erhie** [easytarget/esp32-cam-webserver#292] [Common Issues] [answered]
I want to use the board  Seeed Studio XIAO ESP32S3 Sense .
Are changes necessary for board configuration ?
Thank you for advice
Erhie

**Comment by zz8able** [comment, marked as answer]
You can use this code with Seeed Studio XIAO ESP32S3 Sense.
What you need to do with it is:

disable temprature_sens_read() calls in app_httpd.cpp;

disable flashLED() function and all of its calls everywhere through out the project code files. Or adjust it to XIAO ESP32S3 led pins;

add this pin map to camera_pins.h:
#elif defined(CAMERA_MODEL_XIAO_ESP32S3)
#define PWDN_GPIO_NUM     -1
#define RESET_GPIO_NUM    -1
#define XCLK_GPIO_NUM     10
#define SIOD_GPIO_NUM     40
#define SIOC_GPIO_NUM     39

#define Y9_GPIO_NUM       48
#define Y8_GPIO_NUM       11
#define Y7_GPIO_NUM       12
#define Y6_GPIO_NUM       14
#define Y5_GPIO_NUM       16
#define Y4_GPIO_NUM       18
#define Y3_GPIO_NUM       17
#define Y2_GPIO_NUM       15
#define VSYNC_GPIO_NUM    38
#define HREF_GPIO_NUM     47
#define PCLK_GPIO_NUM     13

define camera model as: 
#define CAMERA_MODEL_XIAO_ESP32S3
in esp32-cam-webserver.ino and myconfig.h (if you use it);

use LED_DISABLE and LAMP_DISABLE flags in myconfig.h or review its usage considering XIAO ESP32S3 board;

Compile code with "ESP32S3 Dev Module" board selected:
psram selected as "OPI PSRAM",
Partition scheme using OTA;

That is basically all what I did, the easy way, to compile and flash project files to Seeed Studio XIAO ESP32S3 Sense.
And it does work in general. There are still some minor bugs with settings though.
